scripts/utils.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+
+"""
+Authors: Tobias Kaiser, Jakob Beckmann
+
+Copyright 2018 ChainSecurity AG
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import sys
+import os
+import re
+import logging
+
+from solc.exceptions import SolcError
+import solc.install
+
+from termcolor import colored
+
+class NoSolidityProject(BaseException):
+    def __init__(self, dir):
+        self.dir = dir
+
+
+class CompilerVersionNotSupported(BaseException):
+    def __init__(self, version, too_old=True):
+        self.version = version
+        self.too_old = too_old
+
+
+class SolidityCompilationException(SolcError):
+    def __init__(self, solc_exception, files):
+        super().__init__(solc_exception.command,
+                         solc_exception.return_code,
+                         solc_exception.stdin_data,
+                         solc_exception.stderr_data,
+                         solc_exception.stdout_data,
+                         solc_exception.message)
+        self.files = files
+
+
+def get_supported_solc_versions():
```

Comment:
Doesn't have to be a function, can just be computed directly in the file.

Reply:
This is existing code from the Travis branch. It is computed directly in the file on [line 133](https://github.com/eth-sri/securify/blob/7b5d077030aa2e4a7e02924713d869be48160457/scripts/utils.py#L133). However, keeping the code in the function is more elegant than having a code block computing the constant in the file imho.